plugins/database/postgresql/postgresql_test.go

```diff
@@ -94,6 +96,53 @@ func TestPostgreSQL_Initialize_ConnURLWithDSNFormat(t *testing.T) {
 	}
 }
 
+// Ensures we can successfully initialize and connect to a CloudSQL database
+// Requires the following:
+// - GOOGLE_APPLICATION_CREDENTIALS either JSON or path to file
+// - CONNECTION_URL to a valid Postgres instance on Google CloudSQL
+func TestPostgreSQL_Initialize_CloudSQL(t *testing.T) {
+	connURL := os.Getenv("CONNECTION_URL")
```

Comment:
Should we skip this since it is an integration test?

Reply:
Yup! Integration tests are now being skipped in CI in [0e7e600](https://github.com/hashicorp/vault/pull/22445/commits/0e7e6001d9a6ce7cab335cfa0feed5b4d97cfbc1)